app/assets/stylesheets/govuk_publishing_components/components/_breadcrumbs.scss

```diff
@@ -20,3 +20,25 @@
 .gem-c-breadcrumbs--inverse .govuk-breadcrumbs__list-item:before {
   border-color: govuk-colour("white");
 }
+
+@include govuk-media-query($until: tablet) {
+
+  .govuk-breadcrumbs__list-item:not(:last-child):not(:first-child) {
+    display: none;
+  }
+
+  .govuk-breadcrumbs__list-item {
+    position: relative;
+    padding-top: 14px;
+    padding-bottom: 14px;
+  }
+
+  .govuk-breadcrumbs__link::after {
+    content: '';
+    position: absolute;
+    top: 0;
+    right: 0;
+    left: 0;
+    bottom: 0;
```

Comment:
Just one small problem with this... if it's the list item with position relative, the after element here gets quite close to the other link, see:

<img width="317" alt="Screenshot 2020-02-14 at 12 29 01" src="https://user-images.githubusercontent.com/861310/74531882-bfe40f00-4f25-11ea-914c-9bf3ca7b635e.png">

However, if you put position relative on the link itself, and change the top and bottom values of the after element to -14px (or whatever) you get this:

<img width="305" alt="Screenshot 2020-02-14 at 12 28 41" src="https://user-images.githubusercontent.com/861310/74531916-d8542980-4f25-11ea-94d5-2962e842ae36.png">

Reply:
have adjusted accordingly as suggested